### DISCUSSION TITLE
Is this just for SvelteKit?

### DISCUSSION THREAD

**Discussion by mikwee** [BattlefieldDuck/xterm-svelte#25] [General]
I'm trying to build a component that utilizes a terminal, but SvelteKit is not needed for that. Would this be helpful?
Thanks.

**Comment by BattlefieldDuck** [comment]
If you are not developing an application with SvelteKit, there’s no need to use this library.

You can use [xterm.js](https://github.com/xtermjs/xterm.js) directly.
